ui/src/services/main/cardTemplate.js

```diff
@@ -20,6 +20,20 @@ class CardTemplateServices {
       return response;
     }
   }
+
+  async verifyCardTemplate(payload) {
+    const response = await apiService.post(
+      "cueSearch/templates/verify/",
+      payload
+    );
+    if (response.success) {
+      return response;
+    } else {
+      return response;
+    }
+  }
```

Comment:
What is this condition doing here? Have you tested UI for all responses - `True` ,`False` & `error`